doc/userguide/devguide/internals/engines/stream/applayertriggerrawreassembly.rst

```diff
@@ -0,0 +1,37 @@
+AppLayerParserTriggerRawStreamReassembly
+----------------------------------------
+
+This is a call for immediate reassembly of raw stream data by the Application Layer in case
+there is a requirement e.g. upon completion of an HTTP request.
```

Comment:
What do you think of adding an explanation as to why this call has to be done from the app-layer side? Suggesting this as this was something asked more than once, so I imagine it might be a common question.